doc-content/enterprise-only/installation/reference-implementations-con.adoc

```diff
@@ -1,15 +1,17 @@
 [id='reference-implementations-con_{context}']
 
-= Available reference implementations
+= {PRODUCT} reference implementations
 
-Three reference implementations are available that you can use as starter applications. They are provided in the *{PRODUCT} {PRODUCT_VERSION_LONG} Reference Implementations* download, available on the {PRODUCT} Software Downloads page in the Red Hat Customer Portal.
+{PRODUCT} provides reference implementations that you can use as starter applications. They are included in the *{PRODUCT} {PRODUCT_VERSION_LONG} Reference Implementations* download, available on the {PRODUCT} Software Downloads page in the Red Hat Customer Portal.
 
 .Employee Rostering reference implementation
 The employee rostering reference implementation enables you to create an application that assigns employees to shifts on various positions in an organization. For example, you can use the application to distribute shifts in a hospital between nurses, guard duty shifts across a number of locations, or shifts on an assembly line between workers.
 
-.High available event-driven decisioning reference implementation
-A reference implementation for high available event-driven decisioning on {OPENSHIFT}
-is available. You can customize this reference implementation to deploy {PRODUCT_DROOLS} engine code that requires stateful processing, including rules developed with complex event processing, in an OpenShift environment. Doing this enables the {DECISION_ENGINE} to process complex event series with high availability.
-
 .Vehicle route planning reference implementation
 The vehicle route planning reference implementation enables you to create an application that solves a vehicle route planning problem with real-world maps, roads, and vehicles delivering goods to locations, each with a specific capacity. For more information, see the README file in the vehicle routing ZIP file, included in the reference implementation download.
+
+.School timetable reference implementation
+The school timetable reference implementation enables you build a REST application on Spring Boot that associates lessons with rooms and time slots and avoids conflicts by considering student and teacher constraints.
```

Comment:
The school timetable reference implementation enables you **to** build a REST application on Spring Boot..

Please add "to" to this sentence.